lesson-01-assignment/lesson-01-assignment.rb

```diff
@@ -0,0 +1,21 @@
+#1
+
+coding_class = CodingClass.find_by({ title: 'Intro to Programming' })
+course_ids = Course.where(coding_class_id: coding_class.id).pluck(:id)

```

Comment:
Almost! There's actually only one `course_id` which you can find by also factoring in `trimester`.